src/LibVLCSharp.Forms/Shared/MediaPlayerElement.xaml.cs

```diff
@@ -17,6 +21,8 @@ public partial class MediaPlayerElement : ContentView
         public MediaPlayerElement()
         {
             InitializeComponent();
+            EqualizerService = new EqualizerUtils();
```

Comment:
This feels like it should be in `PlaybackControls.xaml.cs`. Why only the equalizer logic is in the `MediaPlayerElement` while all other controls logic are in the PlaybackControls?